pyproject.toml

```diff
@@ -44,7 +44,7 @@ azure = [
     "adlfs==2024.4.1",
     "azure-identity>=1.16.0",
     "dbt-sqlserver>=1.3, <1.8",
-    "prefect-azure @ git+https://github.com/Trymzet/prefect-azure@add_keyvault_auth#egg=prefect-azure",
+    "prefect-azure-dyvenia[key_vault]==0.1.0", # Adds Key Vault support to Prefect Azure; from https://github.com/trymzet/prefect-azure/tree/add_keyvault_auth
```

Comment:
Again I'm worried about this change appearing here, because that means you most likely either rebased incorrectly or maybe created the branch without first syncing your local `2.0` branch with remote. But this could be due to a number of things and due to that, this PR can be overriding some other things we're not seeing? For sure this change should not be appearing in this PR.

Reply:
I was talking about the fact that this commit shouldn't be here at all, since this change is already present in 2.0. So if you rebased, this PR would not include this commit, which means the rebase was done incorrectly.

Anyway, just wanted to note this fact in case things go wrong after merging.